Pcap++/src/PcapFilter.cpp

```diff
@@ -348,11 +338,19 @@ void OrFilter::parseToString(std::string& result)
 		result += '(' + innerFilter + ')';
 		if (m_FilterList.back() != *it)
 		{
-			result += " or ";
+			result += m_CompositeDelimiter;
 		}
 	}
 }
 
+AndFilter::AndFilter() : CompositeFilter(" and ") {}
```

Comment:
I also think the operator should be an enum and not a string, probably a protected enum that users can't see, and then `parseToString()` can translate the enum to a string

Reply:
seems to be addressed.